rskj-core/src/main/java/co/rsk/rpc/Web3RskImpl.java

```diff
@@ -102,10 +102,21 @@ public Web3RskImpl(
         this.blockStore = blockStore;
     }
 
-    public void ext_dumpState() {
+    public void ext_dumpState(String account,boolean exportStorageKeys,boolean exportCode) {
         Block bestBlcock = blockStore.getBestBlock();
-        logger.info("Dumping state for block hash {}, block number {}", bestBlcock.getHash(), bestBlcock.getNumber());
-        networkStateExporter.exportStatus(System.getProperty("user.dir") + "/" + "rskdump.json");
+        logger.info("Dumping state for block hash {}, block number {}, account {}",
+                bestBlcock.getHash(), bestBlcock.getNumber(),account);
+        String name = "rskdump";
+        if (account.length()!=0) {
```

Comment:
here you're trying to validate the input, right? I think it can be improved by initializing a  `new RskAddress()` (it has already implemented the validation)